modules/devguide/examples/QueryPrepared.java

```diff
@@ -14,61 +14,48 @@
  * limitations under the License.
  */
 
-package com.couchbase.devguide;
+import java.util.List;
 
 import com.couchbase.client.java.json.JsonObject;
 import com.couchbase.client.java.query.QueryOptions;
 
-import java.util.List;
-
 /**
- * Example of Querying using placeholders with N1QL in Java for the Couchbase Developer Guide.
+ * Example of Querying using placeholders with N1QL in Java for the Couchbase
+ * Developer Guide.
  */
 public class QueryPrepared extends ConnectionBase {
 
     private static final String PLACEHOLDER_STATEMENT = "SELECT airportname FROM `travel-sample` WHERE city=$1 AND type=\"airport\"";
 
     private List<JsonObject> queryCity(String city) {
-        //the placeholder values can be provided as a JSON array (if using $1 syntax)
+        // the placeholder values can be provided as a JSON array (if using $1 syntax)
         // or map-like JSON object (if using $name syntax)
-        JsonObject placeholderValues = JsonObject.create().put("city",city);
+        JsonObject placeholderValues = JsonObject.create().put("city", city);
         return cluster.query("SELECT airportname FROM `travel-sample` WHERE city=$1 AND type=\"airport\"",
-            QueryOptions.queryOptions().parameters(placeholderValues)).rowsAsObject();
+                QueryOptions.queryOptions().parameters(placeholderValues)).rowsAsObject();
     }
 
     @Override
     protected void doWork() {
         JsonObject airport;
 
-        airport = JsonObject.create()
-            .put( "type", "airport")
-            .put("airportname", "Reno International Airport")
-            .put("city", "Reno")
-            .put("country", "United States");
+        airport = JsonObject.create().put("type", "airport").put("airportname", "Reno International Airport")
```

Comment:
ditto (sorry, should have stopped earlier, hadn't realised how many of these there would be, but I'm almost at the end of the file now...)